common/src/main/java/com/netflix/conductor/common/utils/TaskUtils.java

```diff
@@ -28,4 +35,8 @@ public static String removeIterationFromTaskRefName(String referenceTaskName) {
         String[] tokens = referenceTaskName.split(TaskUtils.LOOP_TASK_DELIMITER);
         return tokens.length > 0 ? tokens[0] : referenceTaskName;
     }
+
+    public static WorkflowDef convertToWorkflowDef(Object workflowDef) {
```

Comment:
can't have static objectMapper without screwing up generation of protomapper, so it is here